.github/workflows/benchmark_v2_a10_caller.yml

```diff
@@ -13,6 +13,8 @@ jobs:
     uses: ./.github/workflows/benchmark_v2.yml
     with:
       runner: aws-g5-4xlarge-cache-use1-public-80
```

Comment:
and one minor question, why here it's `runner: aws-g5-4xlarge-cache-use1-public-80` instead of

```
    runs-on:
      group: aws-g5-4xlarge-cache
```

(that is what is used so far in many workflow files ..?)

Reply:
This was a workaround until we got the AMD runner reconfigured as well to be a proper runner group. I'll clean this up in a separate PR.